custom-remote-user-store-management/agent/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+  ~
+  ~  Licensed under the Apache License, Version 2.0 (the "License");
+  ~  you may not use this file except in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~
+  ~  http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.wso2.msf4j</groupId>
+        <artifactId>msf4j-service</artifactId>
+        <version>2.1.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.carbon.remote.user.store</groupId>
+    <artifactId>org.wso2.carbon.remote.user.store.agent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>Remote User Store Agent</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.json.wso2</groupId>
+            <artifactId>json</artifactId>
+            <version>${org.json.wso2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.utils</artifactId>
+            <version>${org.wso2.carbon.utils.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>${mysql-connector-java.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
+            <artifactId>tomcat-jdbc</artifactId>
+            <version>${tomcat-jdbc.version}</version>
+        </dependency>
+    </dependencies>
+
+    <properties>
+        <microservice.mainClass>org.wso2.carbon.remote.user.store.agent.Application</microservice.mainClass>
+        <org.json.wso2.version>2.0.0.wso2v1</org.json.wso2.version>
+        <org.wso2.carbon.utils.version>4.4.11</org.wso2.carbon.utils.version>
+        <mysql-connector-java.version>8.0.13</mysql-connector-java.version>
+        <tomcat-jdbc.version>9.0.14</tomcat-jdbc.version>
+    </properties>
+
+</project>
```

Comment:
Let's add a new line at the EOF.

Reply:
Fixed in 51041691a23b06016ed6fa558504cc92b89c40b3 and 8a4b4fe793b231e0bd48d328c076d4cfd9494c5e